app/models/manageiq/providers/cloud_manager/vm.rb

```diff
@@ -140,6 +140,14 @@ def disassociate_floating_ip(ip_address)
     raw_disassociate_floating_ip(ip_address)
   end
 
+  def service
+    super || orchestration_stack.try(:service)
```

Comment:
you could replace also https://github.com/Ladas/manageiq/blob/32ecbc67632e0525099c1e7177e488125b05987f/app/helpers/vm_helper/textual_summary.rb#L347-L347

also, you should also check a root stack with @record.try(:orchestration_stack).try(:root).try(:service)

otherwise this will not work when vm is deployed as a part of nested stack